### ISSUE TITLE
Trouble with export to csv

### ISSUE THREAD

**Issue by ebdavison** [Pierre-Sassoulas/django-survey#70]
I am trying to export to CSV and am getting an error about NoneType:

```
$ ../manage.py exportresult --survey-id 4 --csv
Traceback (most recent call last):
  File "../manage.py", line 21, in <module>
    main()
  File "../manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "/home/otcomm/otcomm/otcommenv/lib/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/home/otcomm/otcomm/otcommenv/lib/python3.6/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/otcomm/otcomm/otcommenv/lib/python3.6/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/otcomm/otcomm/otcommenv/lib/python3.6/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/home/otcomm/otcomm/otcommenv/lib/python3.6/site-packages/survey/management/commands/exportresult.py", line 78, in handle
    exporter.generate_file()
  File "/home/otcomm/otcomm/otcommenv/lib/python3.6/site-packages/survey/exporter/survey2x.py", line 91, in generate_file
    f.write(self.survey_to_x())
  File "/home/otcomm/otcomm/otcommenv/lib/python3.6/site-packages/survey/exporter/csv/survey2csv.py", line 70, in survey_to_x
    line = Survey2Csv.get_user_line(question_order, response)
  File "/home/otcomm/otcomm/otcommenv/lib/python3.6/site-packages/survey/exporter/csv/survey2csv.py", line 36, in get_user_line
    answers = answer.values
  File "/home/otcomm/otcomm/otcommenv/lib/python3.6/site-packages/survey/models/answer.py", line 51, in values
    if len(self.body) < 3 or self.body[0:3] != "[u'":
TypeError: object of type 'NoneType' has no len()
```

**Comment by Pierre-Sassoulas** [comment]
Hi, thank you for the report!

`if len(self.body) < 3 or self.body[0:3] != "[u'":` should probably become `if self.body and (len(self.body) < 3 or self.body[0:3] != "[u'"):`. You can try to modify `/home/otcomm/otcomm/otcommenv/lib/python3.6/site-packages/survey/models/answer.py` directly while waiting for a fixed release.

In order to add a test case, would it be possible for you dump your database as json ? 
```
python manage.py dumpdata --format json -e contenttypes -e admin -e auth.Permission -e sessions.session -e sites.site --natural-foreign --indent 1 -o issue70.json
```
In your survey 4, there should be an example of Answer where body == None that could be added to the test cases.

**Comment by ebdavison** [comment]
Here is a cleaned up export with just the survey and the answers.  the last question has an empty response.
[issue70.zip](https://github.com/Pierre-Sassoulas/django-survey/files/4156718/issue70.zip)

**Comment by Pierre-Sassoulas** [comment]
Thank you for the test data, I released 1.3.14 with a fix for this. It also has german translation.
